static/_redirects

```diff
@@ -1,5 +1,11 @@
 # Redirects
 
+/support/how-to/shut-down-a-cloud-server/  /how-to/shut-down-a-rackspace-cloud-server/  404
+/how-to/shut-down-a-rackspace-cloud-server/   /support/how-to/cloud-servers/shut-down-a-rackspace-cloud-server/  301 
```

Comment:
The last URL should be `/support/how-to/shut-down-a-rackspace-cloud-server/` with no "cloud-servers" in there. We don't put the product name in the URL

Same thing for the other two